paparazzi/src/main/java/app/cash/paparazzi/internal/SessionParamsBuilder.kt

```diff
@@ -28,10 +29,12 @@ import com.android.ide.common.rendering.api.SessionParams.RenderingMode
 import com.android.ide.common.resources.ResourceResolver
 import com.android.ide.common.resources.ResourceValueMap
 import com.android.ide.common.resources.deprecated.ResourceRepository
+import com.android.resources.LayoutDirection
 import com.android.resources.ResourceType
 
 /** Creates [SessionParams] objects. */
 internal data class SessionParamsBuilder(
+  private val bridge: Bridge,
```

Comment:
Definitely don't want to pass Bridge into the param builder, but there is a static method Bridge.isLocaleRtl that seems to do the same.

Reply:
Thanks for pointing it out, fixed!